pylintrc

```diff
@@ -179,4 +180,4 @@ int-import-graph =
 [EXCEPTIONS]
 overgeneral-exceptions = Exception
 
-# 3dd01da1adbb2cac2fbed3cbc578b3f647b792c7
```

Comment:
With this tool in place, a pylintrc file is not a source file, it's generated from a central pylintrc, and a local pylintrc_tweaks.  If you edit the pylintrc file directly, and then later someone overwrites it with "edx_lint write pylintrc", then your edited changes will be lost.  The hash allows edx_lint to detect that the file had been edited since generation, and keep the edited file for you to resolve by hand.